src/components/SectionHeader.tsx

```diff
@@ -10,14 +10,21 @@ type Props = {
 };
 
 export const SectionHeader = ({ title, onPress }: Props) => {
-  const innerComponent = (
-    <Title accessibilityLabel={`${title} (Überschrift) (Taste)`}>{title}</Title>
-  );
+  const innerComponent = <Title>{title}</Title>;
 
   return (
     <>
       <TitleContainer>
-        {onPress ? <Touchable onPress={onPress}>{innerComponent}</Touchable> : innerComponent}
+        {onPress ? (
+          <Touchable
+            onPress={onPress}
+            accessibilityLabel={`${consts.a11yLabel.heading} ${consts.a11yLabel.button}(${title})`}
+          >
+            {innerComponent}
+          </Touchable>
+        ) : (
+          innerComponent
```

Comment:
now this does no longer have an accessibilty label. please find a way that works in both cases.

Reply:
yes you're right! but for some reason it still woks just fine .. have you tested it on you device ?

Reply:
so we tested on a correct place that the inner component is missing the (Überschrift).
we need a title with überschrift for the `Title` and a title with überschrift and taste for the `Touchable`. could it be a problem with having two nested `accessibilityLabel`?

Reply:
Ok, the new version seems to work fine.
now only the inner component has an accessibility label, and if there is an `onPress` then the `(Taste)` will be added to it.